images/kube-webhook-certgen/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#Builder stage
+FROM ppc64le/golang:1.25.3 AS builder
+
+ARG TARGETARCH=ppc64le
+ARG GOLANG_VERSION=1.25.3
```

Comment:
Not sure if we're using this ARG anywhere.. Please feel free to remove if it's not necessary.
```suggestion
```